http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -403,12 +403,16 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
 
       _ <- if (hasStopPermission) F.unit else F.raiseError[Unit](ForbiddenError(userInfo.userEmail))
       // throw 409 if the cluster is not stoppable
-      _ <- if (runtime.status.isStoppable) F.unit
-      else
+
+      _ <- if (runtime.status.isStopping) F.unit
```

Comment:
shouldn't `Stopped` status in this branch as well?

Reply:
Oh yeah, +1

Reply:
oh i think i misunderstood your previous comment, will add it back